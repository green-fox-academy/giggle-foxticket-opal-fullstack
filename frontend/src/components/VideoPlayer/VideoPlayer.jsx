import React from 'react';
import ReactPlayer from 'react-player/youtube';
import './VideoPlayer.styles.sass';

export default function () {
  return (
    <div className="video">
      <ReactPlayer
        url="https://www.youtube.com/watch?v=KIQ_YEKgVr0"
        controls={true}
      />
    </div>
  );
}
