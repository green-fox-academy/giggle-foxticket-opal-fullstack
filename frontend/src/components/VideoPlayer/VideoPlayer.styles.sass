.video 
  display: flex
  justify-content: center
  margin: 0% 5% 5% 5%
