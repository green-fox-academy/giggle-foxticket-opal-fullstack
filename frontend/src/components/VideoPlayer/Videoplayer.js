import React from 'react'
import ReactPlayer from 'react-player/youtube'
import './VideoPlayer.styles.sass';

export default function () {
  return (
    <div className="video">
      <ReactPlayer url='https://www.youtube.com/watch?v=qREKP9oijWI' controls={true} />
    </div>
  )
}

