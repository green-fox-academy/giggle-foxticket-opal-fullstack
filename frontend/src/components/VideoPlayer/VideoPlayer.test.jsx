import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './VideoPlayer';

it('renders correctly', () => {
  const tree = renderer.create(<VideoPlayer />).toJSON();
  expect(tree).toMatchSnapshot();
});
