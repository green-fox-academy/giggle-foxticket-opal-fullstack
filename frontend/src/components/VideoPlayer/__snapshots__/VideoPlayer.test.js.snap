// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="video"
>
  <div
    style={
      Object {
        "height": "360px",
        "width": "640px",
      }
    }
  >
    <div
      style={
        Object {
          "display": undefined,
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <div />
    </div>
  </div>
</div>
`;
